qiita_pet/templates/change_lost_pass.html

```diff
@@ -8,7 +8,7 @@
 {%end%}
 
 {% block content %}
-  {% if not message %}
+  {% if display_body %}
```

Comment:
This can be done much cleaner as 
```
if maintenance is None
```
and also would not require a new variable to be created.